book/src/offline-signing/README.md

```diff
@@ -0,0 +1,78 @@
+# Offline Transaction Signing
+
+Some security models require keeping signing keys, and thus the signing
+process, separated from transaction creation and network broadcast. Examples
+include:
+  * Collecting signatures from geographically disparate signers in a
```

Comment:
If we list this as an example use case, I think we should have instructions for it on this page. I've never personally seen what the experience of collecting multiple disparate signatures looks like with our CLI tools, it'd be interesting to see!

Reply:
'tis here https://github.com/solana-labs/solana/blob/master/book/src/api-reference/cli.md#multiple-witnesses though that whole page could use some work

Reply:
Cool, how about we link to it?